src/core/ext/filters/http/client/http_client_filter.cc

```diff
@@ -465,16 +465,16 @@ static void hc_start_transport_stream_op_batch(
 }
 
 /* Constructor for call_data */
-static grpc_error* init_call_elem(grpc_call_element* elem,
-                                  const grpc_call_element_args* args) {
+static grpc_error* hc_init_call_elem(grpc_call_element* elem,
```

Comment:
Suggest renaming to `http_client_*`. `hc` sounds more like health checking to me.

Reply:
good idea